auto_submit/lib/service/github_service.dart

```diff
@@ -148,26 +148,44 @@ class GithubService {
     List<PullRequestFile> originalPullRequestFiles = await getPullRequestFiles(repositorySlug, revert);
     List<PullRequestFile> currentPullRequestFiles = await getPullRequestFiles(repositorySlug, current);
 
-    return _validateFileSetsAreEqual(originalPullRequestFiles, currentPullRequestFiles);
+    return validateFileSetsAreEqual(originalPullRequestFiles, currentPullRequestFiles);
   }
 
   /// Validate that each pull request has the same number of files and that the
   /// file names match. This must be the case in order to process the revert.
-  bool _validateFileSetsAreEqual(
-    List<PullRequestFile> revertPullRequestFiles,
-    List<PullRequestFile> currentPullRequestFiles,
-  ) {
-    List<String?> revertFileNames = [];
-    List<String?> currentFileNames = [];
-
-    for (PullRequestFile element in revertPullRequestFiles) {
+  bool validateFileSetsAreEqual(List<PullRequestFile> changeList1, List<PullRequestFile> changeList2) {
```

Comment:
Should we make the parameters more descriptive?

Reply:
Yes. Let me make them more indicative of what they represent.